### DISCUSSION TITLE
Updating user's system fields (email)

### DISCUSSION THREAD

**Discussion by synapse** [pocketbase/pocketbase#1032] [Q&A] [answered]
I'm trying to update the current user's email address, thus using a user JWT token, and it does not work. Works flawlessly though with any other **non system** field (also works as expected from the admin dashboard).
Is there another way to let users change their email address or is this a bug?

![SCR-20221115-fk0](https://user-images.githubusercontent.com/71137/201893959-6dd32a30-55f6-46d9-8086-228e19743002.png)

**Comment by ganigeorgiev** [comment, marked as answer]
@synapse The user email address can be changed directly only by admins and the users who has "Manage API rule" access.
Regular user can change their email by sending `POST /api/collections/users/request-email-change` request.

**Reply by synapse** [reply]
Ah perfect, thank you!
